simulation/executor/simulate.go

```diff
@@ -60,17 +41,21 @@ const AverageBlockTime = 6 * time.Second
 func SimulateFromSeed(
 	tb testing.TB,
 	w io.Writer,
-	app simtypes.App,
+	appCreator simtypes.AppCreator,
 	initFunctions InitFunctions,
-	actions []simtypes.ActionsWithMetadata,
 	config Config,
-) (stopEarly bool, err error) {
+) (lastCommitId storetypes.CommitID, stopEarly bool, err error) {
 	// in case we have to end early, don't os.Exit so that we can run cleanup code.
 	// TODO: Understand exit pattern, this is so screwed up. Then delete ^
 
+	legacyInvariantPeriod := uint(10) // TODO: Make a better answer of what to do here, at minimum put into config
```

Comment:
This todo should just get handled with more property check work, and being able to set this to 0